docs/content/node-operation/node-setup.mdx

```diff
@@ -72,6 +72,18 @@ Flow also provides `systemd` service and unit files as a template for installati
   those are good choices.
 </Callout>
 
```

Comment:
Shall we add a newline here? Git renders this without any section separation (screenshot): 
<img width="882" alt="Screen Shot 2021-07-26 at 10 31 17 AM" src="https://user-images.githubusercontent.com/26322303/127032950-739fa5cd-fb95-488f-841e-f4cf14c147c7.png">

```suggestion
### Time synchronization

```